### DISCUSSION TITLE
2.1.3

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#26289] [General]
The Apache Superset team is pleased to announce that Superset
2.1.3 has just been released.

Apache Superset is a modern, enterprise-ready business intelligence web application

The official source release:

https://www.apache.org/dist/superset/2.1.3

The PyPI package:

https://pypi.org/project/apache-superset/

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/2.1.3'>2.1.3</a>.</em>

**Comment by MaratIR** [comment]
why is rc1 in the archive?

**Reply by eschutho** [reply]
HI @MaratIR. Which archive are you referring to?
